canonical-web-and-design/maas-ui
# Allow script results to be suppressed and unsuppressed.

## Done

- Allow script results to be suppressed/unsuppressed.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit machine details testing for a machine with a suppressable script result e.g. http://0.0.0.0:8400/MAAS/r/machine/feeaq4/tests
- Ensure the scriptResult can be suppressed and unsuppressed by toggling the checkbox

## Fixes

Fixes: canonical-web-and-design/MAAS-squad#2207

## Launchpad issue

Related Launchpad maas issue in the form `lp#number`.

## Backports

In general, please propose fixes against *master* rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in master.
